### ISSUE TITLE
IITC on Chrome - No longer works after IOS update

### ISSUE THREAD

**Issue by summoner12** [iitc-project/ingress-intel-total-conversion#1184]
I ran an update on my computer and have now lost IITC. The update I made was a security update for El Capitan ver. 10.11.6. 

Has anyone else had this issue or have ideas on how to fix it. 

When the issue happened I restored my computer back a day before the update and IITC worked just fine. Then, not realizing it, my computer did the update again and IITC failed again. I don't really want to restore again since it took 7 hours last time (old iMac :( ) 

Any info would be great.
Thanks!

**Comment by mxxcon** [comment]
Define "no longer works". You have not provided any error message or any details that would help troubleshooting or fixing this issue.

**Comment by summoner12** [comment]
Hello mxxcon. 

What I mean by not working; stock intel loads IITC does not create an overlay. 

I meant to reply and close out this issue as I fixed it a day after I made my OP.

I ended up fully updating my imac to sierra and still wasn't getting IITC loaded when going to ingress intel in chrome. So I removed the tampermonkey extension and reinstalled it. IITC overlay loaded over stock intel map at that time.

Maybe that is how simple of a repair it was before having to do all of the IOS updates? but now it's working, so I am happy again.

Thank you for responding!
